charts/pravega/README.md

```diff
@@ -42,6 +42,13 @@ helm install [RELEASE_NAME] pravega/pravega --version=[VERSION] --set zookeeperU
 ```
 Based on the cluster flavours selected,segmentstore memory requirements needs to be adjusted.
 
+## Updating the chart
```

Comment:
Again, we want to **upgrade Pravega** here so lets call this section "Upgrading Pravega" instead of the current title.
Lets also add a link to the upgrade documentation, if users want to understand details of the upgrade process.

Reply:
Done